Minor Fix: Updated Loki Otlp Ingest Configuration (#12648)
grafana/loki: docs/sources/send-data/otel/_index.md

When logs are ingested by Loki using an OpenTelemetry protocol (OTLP) ingestion endpoint, some of the data is stored as [Structured Metadata]({{< relref "../../get-started/labels/structured-metadata" >}}).

> *Make sure to set the `allow_structured_metadata` to `true` within your Loki config file. Otherwise, Loki will reject the log payload as malformed.*

Reviewer: Our style guide says not to use single asterisks for emphasis.
```suggestion
You must set `allow_structured_metadata` to `true` within your Loki config file. Otherwise, Loki will reject the log payload as malformed.
```
